GoogleForCreators/web-stories-wp
# Remove all illegal characters from slug

## Summary
Remove all illegal characters from the permalink with blurring, similar behaviour to what WP is doing generally.


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Open the Permalink panel in the Document tab.
2. Type in a URL with illegal characters, e.g.`hello!:;world`.
3. Tab out of the input.
4. Verify that all illegal characters have been removed.

### UAT







This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #6480
